recipe/meta.yaml

```diff
@@ -20,7 +19,8 @@ requirements:
   host:
     - python
     - pip
-    - setuptools
+    - setuptools >=61.2.0
```

Comment:
these setuptools requirements seem to have come from the python-lsp-server repo, which is copied in

Reply:
addressed